pyvista/core/_validation/check.py

```diff
@@ -374,6 +374,13 @@ def check_nonnegative(array: _ArrayLikeOrScalar[NumberType], /, *, name: str = "
     check_greater_than(array, 0, strict=False, name=name)
 
 
+def _validate_real_finite_value(scalar, name='Value'):
```

Comment:
This used to `check_finite` but was removed, the name should reflect this
```suggestion
def _validate_real_value(scalar, name='Value'):
```